charts/host-check/values.schema.json

```
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "http://cisco.com/xrd/helm",
+  "title": "host-check Helm Chart Values",
+  "description": "Configuration values for the host-check helm charts",
+  "type": "object",
+  "properties": {
+    "image": {
+      "description": "Container image config",
+      "type": "object",
+      "properties": {
+        "repository": {
+          "description": "Repository to pull the image from (required)",
+          "type": "string"
+        },
+        "tag": {
+          "description": "Image tag to pull from the repository (required)",
+          "type": "string"
+        },
+        "pullPolicy": {
+          "description": "Standard Kubernetes image pull policy",
+          "type": "string",
+          "default": "Always",
+          "enum": [
+            "Always",
+            "IfNotPresent",
+            "Never"
+          ]
+        },
+        "pullSecrets": {
+          "description": "Standard Kuberenetes pod imagePullSecrets array",
+          "type": "array"
+        }
+      },
+      "required": [
+        "repository",
+        "tag"
+      ],
+      "additionalProperties": false
+    },
+    "targetPlatforms": {
+      "description": "List of platforms to run the host-check on",
```

```suggestion
      "description": "List of platforms to run host-check on",
```

or "the host-check application"